Documentation/specs/corehost.md

```diff
@@ -93,6 +93,32 @@ The runtime is located by searching the following paths in order, where `APP_BAS
     * `%LocalAppData%\dotnet\runtime\coreclr\LIBCORECLR`
     * `%ProgramFiles%\dotnet\runtime\coreclr\LIBCORECLR` [2]
 
+
+## CoreHost Servicing
+
+For **RC** milestone, the servicing for the host is tied to the `dotnet/cli` tools i.e., the app developer would have to pull the `corehost.exe` and `hostpolicy.dll` from the appropriate CLI binary drop and replace it in their app or re-publish their app with a serviced CLI.
+
+For **RTM**:
+* We will version the host independently as NuGet packages.
+* WU based servicing
+   * The host will leave a breadcrumb at a known (TBD) location about its own version for WU.
+      * Example: `$BREADCRUMB/COREHOST_PACKAGE_NAME/COREHOST_PACKAGE_VERSION.txt`
+* General servicing
+   * The servicing process would pull in the right NuGet package and place it in:
+      * `$DOTNET_SERVICING/COREHOST_PACKAGE_NAME/COREHOST_PACKAGE_VERSION/<SERVICED_PACKAGE_CONTENTS>`
```

Comment:
The package will carry both, as the shim by design is not serviceable and the packages provide a way for the user to update the shim. I will include this in the spec.